extensions/ql-vscode/src/history-item-label-provider.ts

```diff
@@ -74,8 +74,7 @@ export class HistoryItemLabelProvider {
       // Return the number of repositories queried if available. Otherwise, use the controller repository name.
       d: numRepositoriesQueried ? numRepositoriesLabel : `${item.remoteQuery.controllerRepository.owner}/${item.remoteQuery.controllerRepository.name}`,
 
-      // There is no synchronous way to get the results count.
-      r: '',
+      r: item.resultCount === undefined ? '' : `(${item.resultCount} results)`,
```

Comment:
Elsewhere we've been careful with pluralization (`result` if 1 result, `results` if more than one), shall we do the same here?

Reply:
Makes sense. I've extracted a `pluralize` method here: https://github.com/github/vscode-codeql/pull/1432/commits/0577f762df76b83630097e973c81c8c470c3fc15

Let me know what you think.